macos/mac_os_resource_snd.ksy

```diff
@@ -0,0 +1,200 @@
+meta:
+  id: mac_os_resource_snd
+  endian: be
+  license: MIT
+  xref:
+    wikidata: Q7564684
+enums:
```

Comment:
Cccording to the [KSY Style Guide](https://doc.kaitai.io/ksy_style_guide.html#type), `enums` should be present after `doc-ref` and `seq`. In most cases, it's listed at the very end.

Reply:
Moved enums below